force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -7,11 +7,20 @@ public class CollectionsTasks {
     List<Integer> listWithZerosAndUnits
   ) {
     integer countZeros = 0;
-    for (Integer element : listWithZerosAndUnits) {
-      if (element == 0) {
-        countZeros++;
-      }
+    // no IFs solution
+    integer i = 0; //use for iteration in while loop
+    listWithZerosAndUnits.sort(); // sorted asc so that zeros are at the beginning of the list
+    system.debug('sorted: ' + listWithZerosAndUnits);
+    // iterate through the sorted list
+    while (i < listWithZerosAndUnits.size() && listWithZerosAndUnits[i] == 0) {
```

Comment:
👍 

not bad 😃 I didn't think of this solution, though it works too.

What I had in mind was using the fact it's a bit list, so it only has zeroes and ones. But your solution counts

Reply:
added also sum up bits solution